eng/scripts/Get-ApiViewCommandLine.ps1

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+# Usage: Get-ApiViewCommandLine.ps1 .\azure-sdk-for-cpp azure-identity
+# Or: ..\Get-ApiViewCommandLine.ps1 . azure-core
+# Or: Get-ApiViewCommandLine.ps1 ..\.. azure-security-attestation
+# Or: c:\src\azure-sdk-for-cpp\eng\scripts\Get-ApiViewCommandLine.ps1 c:\src\azure-sdk-for-cpp azure-identity
+
+param([String]$RepoPath, [String]$LibName)
```

Comment:
Most of the other languages tie into their common discovery mechanism (i.e. https://github.com/Azure/azure-sdk-for-cpp/blob/main/eng/scripts/Language-Settings.ps1#L15) should we use that here and perhaps add extra information like the header files as another property?

Reply:
Hmm, this is interesting, especially from the point of having all the project information in one structure, but it would serve different purposes, and will be filled in differently. But the result would be easy to use.
But I'd say, let's get it to work in the current straightforward simple form, given my limited PowerShell knowledge (and slow PS development progress as a result), and given that Praveen will only have 1-2 days from now to integrate, until the next window comes up in about a month from now.
I am not rejecting the idea, but with all these factors, debugging complexity when making it all at once, I think the best way to go is to get it working, and then refactoring.

Reply:
Sounds good. I'm just trying to get most of common project properties going through the same place instead of having the logic spread out in various places.